{
  NullOn303 function=new NullOn303();
  RuntimeException exception=new RuntimeException();
  try {
    function.create(exception);
  }
 catch (  Exception ex) {
    assertEquals(ex,exception);
  }
}
